python/py-rawpy/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+name                py-rawpy
+version             0.17.2
+revision            0
+license             MIT
+
+github.setup        letmaik rawpy v0.17.2
```

Comment:
this isn't the correct usage of the `github.setup` line, the `v` version prefix should be after the actual version number as in:
```github.setup        letmaik rawpy 0.17.2 v```

Reply:
I thought the preference for MacPorts was be build everything ourselves. Besides, for the Mac version, they include their own version of libraw (which is also the reason for below).

Reply:
OK, I reconsidered, and added a dedicated email address.